drivers/stepper/Kconfig

```diff
@@ -24,6 +24,11 @@ config STEPPER_SHELL
 	help
 	  Enable stepper shell for testing.
 
+config STEP_DIR_STEPPER
+	bool
```

Comment:
Can you check the indentation over here it looks a bit off

Reply:
![image](https://github.com/user-attachments/assets/6f0330a9-9be6-4cfe-8bd2-b9cd4f9b6b82)
Looks fine to me.

Reply:
![image](https://github.com/user-attachments/assets/c4bfceb2-7336-4254-bcab-f012d6635ef8)

Reply:
@jilaypandya something with your fonts? looks fine here
![x](https://github.com/user-attachments/assets/c6d20f0c-50e2-4d8a-ab00-71fc784ffddb)

Reply:
yep my bad, it seems github mobile renders it a bit different

Reply:
yep, it seems that Github App renders it a bit differently.